build.gradle.kts

```diff
@@ -9,6 +9,16 @@ plugins {
     id("org.jetbrains.compose") apply false
 }
 
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath("dev.icerock.moko:resources-generator:0.20.1")
+    }
+}
+
```

Comment:
let's use same way as up - with plugins block.
```
plugins {
  id("dev.icerock.mobile.multiplatform-resources") apply false
}
```

and in settings.gradle.kts add info about version